api_documentation.md

```
@@ -0,0 +1,945 @@
+*   **Description:** Initiates termination tasks for one or more instances within a specific project.
+*   **Authentication:** Required. Pass the API key in the `apikey` header.
+*   **Request Body:** JSON object with project and instance identifiers.
+    ```json
+    // Expected request structure based on successful test
+    {
+      "owner_id": 1, // Required: Owner ID of the project/instances
+      "project_name": "your-project-name", // Required: Name of the project
+      "instance_names": ["instance-name-1", "instance-name-2"] // Required: Array of instance names to terminate
+    }
+    ```
+    *   **Note:** The codebase (`internal/types/instance.go`) defines `DeleteInstancesRequest` with `InstanceIDs []uint`. However, testing shows the endpoint requires `instance_names []string` for successful operation. This documentation reflects the empirically validated behavior.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)